src/test/resources/features/15-attachment/attachments.feature

```diff
@@ -0,0 +1,18 @@
+ Feature: Attachments Management
+
```

Comment:
Please add a scenario where the camera icon is used

Reply:
Am working on this and updating the pr ASP